portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -11,6 +11,48 @@ definition {
 		User.firstLoginPG();
 	}
 
+	@description = "This test checks 7.3.x Theme Layout can be applied to home page."
+	@priority = "5"
+	test ApplyThemeToSinglePage {
+		property portal.acceptance = "false";
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#ApplyThemeToSinglePage";
+
+		ProductMenu.gotoPortlet(
+			category = "Apps",
+			panel = "Control Panel",
+			portlet = "App Manager");
+
+		AppManager.uploadAppCP(
+			appName = "test3-theme",
+			warFileName = "test3-theme.war");
+
```

Comment:
This is ok but ideally don't want to rely on App Manager UI in case it breaks. Look into refactoring FragmentsDeploy.deployZipOnLiferayPortal to make it more general for deploying dependency files.

- change macro to Dependencies.deployDependency
- refactor so its generalized for all dependencies
- change theme war to test-theme-7.3.war

Reply:
Notes to self: 
- Will need to update the test name and some other fixes to the theme war in a follow up ticket for other tests.

- Will want to rename to 73x-theme and use the 731-theme.war or rebuild. It seemed to be more stable during 731 manual testing and the name will better match.

- May want to include ftl and js edits and turn this into a smoke test.

Reply:
FragmentsDeploy did not work with .war files. Investigate how to deploy war file.